packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app/define-callback/react-native/definescheme.md

```diff
@@ -0,0 +1,3 @@
+<GuideLink link=”../ios”>Set up  iOS redirect scheme</GuideLink>
```

Comment:
removed extra space in front if iOS

<GuideLink link=”../ios”>Set up iOS redirect scheme</GuideLink>